### PR TITLE
Update to SwiftMailer 6.x, Fix #48

### LINKED ISSUE #48: SwiftMailer 6
Hi, 
I need to use SwiftMailer ^6.0 in my project but your bundle requires the 5.0 version — is this compatible ? In this case, could you change the requirements for something like ^5.0 || ^6.0 ?

Thanks a lot !

### PR DESCRIPTION
**Pull request by tchapi**
Update for SwiftMailer 6.0
Some BC

### CODE REVIEW

**Comment by tchapi** [comment]
All tests pass on PHP > `7.x` which is the target so I guess it's good.
I would be grateful if you could tag a specific release for SM 6.0 in that case

Do not hesitate if you think some more work is needed on this PR

BR

**Comment by tchapi** [comment]
Hi, any update ?
Thanks

File: Service/MailgunTransport.php [hunk 1]
```diff
@@ -202,4 +203,12 @@ protected function getDomain(Swift_Mime_Message $message)
 
         return $this->domain;
     }
+
+    /**
+     * Not used.
```
Comment (nickvanderzwet) [inline Service/MailgunTransport.php]: If it's not used it can be removed?

Comment (tchapi) [reply Service/MailgunTransport.php]: It's not used but it's necessary for the class to comply to the interface

**Comment by man-andy** [comment]
Any update on this PR?

**Comment by pculka** [comment]
it fails while using PHP < 7. Guess it's not a problem though when we want to use Swiftmailer 6?

**Comment by tchapi** [comment]
SM 6.0 is `PHP > 7.*` so yes, that's expected.

**Comment by pculka** [comment]
sooo, what's the problem? with a merge? :s

**Comment by tchapi** [comment]
Which problem are you talking about ? Travis complains because it has tests for PHP < 7 so these will not pass, but that's not an issue. As for this PR, it is perfectly mergeable, so we're just waiting on the repo maintainer to merge and make a release when he finds time and if he's ok with it.

**Comment by pculka** [comment]
yes, that's the problem I talk about... that the repo maintainer needs to merge and make a release :D

**Comment by tehplague** [comment]
Performed the merge and bumped the bundle's version number to 1.0 since it breaks BC with PHP < 7 and Swiftmailer < 6

**Comment by tchapi** [comment]
🎉

**Pull request merged**